pyomo/contrib/mindtpy/config_options.py

```diff
@@ -208,6 +208,14 @@ def _get_MindtPy_config():
         description='Partition objective with the sum of nonlinear terms using epigraph reformulation.',
         domain=bool
     ))
+    CONFIG.declare('quadratic_strategy', ConfigValue(
+        default=0,
+        domain=In([0, 1, 2]),
+        description='How to treat the quadratic terms in MINLP.'
+                    '0 : treat as nonlinear terms'
+                    '1 : only keep quadratic terms in objective function'
+                    '2 : keep quadratic terms in objective function and constraints',
```

Comment:
I would suggest:
'1 : only use quadratic terms in objective function directly in main problem'
                    '2 : use quadratic terms in objective function and constraints in main problem',